### PR TITLE
MOBILE-1040 Restore Form when rotate

### PR DESCRIPTION
**Pull request by victorlaerte**

### CODE REVIEW

**Comment by victorlaerte** [comment]
Depends on #595

File: android/samples/test-app/src/main/res/layout/ddm_form.xml [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     >
 
-    <com.liferay.mobile.screens.thingscreenlet.screens.ThingScreenlet
+    <com.liferay.mobile.screens.ddm.form.DDMFormScreenlet
         android:id="@+id/ddm_form_screenlet"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
+        app:formInstanceId="37105"
```
Comment (dgarciasarai) [inline android/samples/test-app/src/main/res/layout/ddm_form.xml]: this id should be in server_context file

Comment (victorlaerte) [reply android/samples/test-app/src/main/res/layout/ddm_form.xml]: What about an app with more than one Form?

Comment (victorlaerte) [reply android/samples/test-app/src/main/res/layout/ddm_form.xml]: you mean, only for test app ?

Comment (dgarciasarai) [reply android/samples/test-app/src/main/res/layout/ddm_form.xml]: I mean for test-app :) All our ids are in server_context file, so if you want to try one screenlet, you go there and edit the id

**Comment by dgarciasarai** [comment]
🎉

**Comment by victorg1991** [comment]
It has conflicts now :S

**Comment by victorlaerte** [comment]
@victorg1991 
not anymore

**Pull request merged**